packages/open-next/src/adapters/cache.ts

```diff
@@ -517,11 +517,17 @@ export default class S3Cache {
 
   private async deleteS3Objects(key: string) {
     try {
-      const regex = new RegExp(`\.(json|rsc|html|body|meta|fetch|redirect)$`);
+      const regexString = `\.(json|rsc|html|body|meta|fetch|redirect)$`;
+      const regex = new RegExp(regexString);
       const s3Keys = (await this.listS3Object(key)).filter(
         (key) => key && regex.test(key),
       );
 
+      if (s3Keys.length === 0) {
+        warn(`No s3 keys matching ${regexString} found for ${key}`);
```

Comment:
You can keep the origin `regex`, it should print out the regex string b/c the regex implements `toString()`

Reply:
Our explicit vs implicit meet again, im gonna stick with explicit since it has always served me well from an understandability standpoint. If you feel strongly, we can poll some others.

I will leave this unresolved as a reminder, you may resolve or not as you see fit.

Reply:
imo `const regex = new RegExp(`\.(json|rsc|html|body|meta|fetch|redirect)$`);` is cleaner b/c it's 1 less line and it doesn't have the `\\.` which can be strange if you're not too familiar w/ regex.

You can still call the toString explicitly, eg `warn(`No s3 keys matching ${regex.toString()} found for ${key}`);`

Reply:
On that note, there's not any dynamic value in `new Regex`, so it coulda have been the RegExp literal, eg `/\.(json|...)$`

Reply:
Ran some tests, and the single backslash was correct, but since (I think) you read `apostrophe` where was `backtick`, went to the drawing board and attempted a clear, consise, explicit compromise that no one can get confused over.  Resolving here, post another convo if you disagree.

Reply:
Thanks! LGTM. I got confused, I thought the original was using the regex literal.
When using `new RegExp`, you have to double escape the `\.`, eg
```
const regexString = `\.(json|rsc|html|body|meta|fetch|redirect)$`;
const regex = new RegExp(regexString);

console.log('should be true: ', regex.test('hello.json'))
console.log('should be false: ', regex.test('hello,json')) // this returns true because the regex is "." instead of "\."
```

Reply:
Went even further. ;)

Reply:
lol thanks! It's not for me, it's going to be for everyone using open-next